src/OgHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\OgHelper.
+ */
+
+namespace Drupal\og;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Field\FieldException;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+
+/**
+ * OG helper methods.
+ */
+class OgHelper {
```

Comment:
maybe this should be called `OgGroupAudience` or `OgGroupAudienceHelper`? - I think it would make its content clearer